test/edu/wright/cs/raiderplanner/model/RequirementTest.java

```diff
@@ -21,19 +21,224 @@
 
 package edu.wright.cs.raiderplanner.model;
 
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import edu.wright.cs.raiderplanner.controller.MenuController;
+
 import org.junit.jupiter.api.Test;
 
 /**
  * Created by bijan on 06/05/2017.
  */
+
 public class RequirementTest {
 
+	/**
+	 * Test of isComplete method, of class Requirement.
+	 */
```

Comment:
A better way to state this would be: `Method under test: {@link Requirement.isComplete()}`

That allows someone viewing the method Javadoc of the test to see a link back to the actual method being tested, which is rather convenient. Please update this and the other occurrences.